dev-python/nuitka/nuitka-0.6.5.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6 python3_7)
+inherit distutils-r1 eutils
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="https://www.nuitka.net"
+SRC_URI="https://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+RDEPEND="${DEPEND}
+	dev-python/appdirs[${PYTHON_USEDEP}]
+	>dev-util/scons-3.0.5[${PYTHON_USEDEP}]
+	"
```

Comment:
This last indent is unnecessary, didnt repoman complain about it? Ebuild looks otherwise :)

Reply:
corrected.